video/out/gl_wayland.c

```diff
@@ -127,6 +127,14 @@ static bool egl_create_context(struct vo_wayland_state *wl,
     return true;
 }
 
+static bool frame_pending = false;
```

Comment:
should I drop this into vo_wayland_state, or what would be technically correct for gl_wayland?